### ISSUE TITLE
Users download courses and take offline

### ISSUE THREAD

**Issue by leomaxi** [open-learning-exchange/take-home#43]
Change take-home UI

**Comment by timecomplexity** [comment]
Since the title was changed here, does this refer to _just_ the ability to download and take courses offline, or do we also need to update the UI?
I would assume that if the user "updates" the app, we would then want it to update the offline courses as well, just like the library items. I just want to make sure I'm on the same page for this issue. :)

**Comment by leomaxi** [comment]
No. It is solely about "Users download courses and take offline". That will directly or indirectly affect the looks.. But not necessarily.

**Comment by dogi** [comment]
@leonardmensah @zeivhann @kylemathias just make sure that you are looking into the new course structure ... > 0.12.104
